cloudify_aws/sqs/tests/test_queue.py

```diff
@@ -114,7 +118,7 @@ def test_create_raises_UnknownServiceError(self):
             queue.create(ctx=_ctx, resource_config=None, iface=None)
 
         self.assertEqual(
-            str(error.exception),
+            text_type(error.exception),
```

Comment:
Same question. No change is necessary, just curious.

Reply:
Same as above